src/styles/_buttons.scss

```diff
@@ -1,92 +0,0 @@
-.btn, .btn-primary, .btn-secondary, .btn-primary-enterprise, .btn-secondary-enterprise, .btn-secondary-enterprise-ico {
-  font-weight: 700;
-  line-height: 1.5;
-  padding: 11px 0;
-  border-radius: 3px;
```

Comment:
Are we 100% confident this file is not used anymore?

Reply:
It seems to me that the global nav is taking over the new style. I didn't find any issues with removing the file. Dan has approved so I assume the style looks fine after the removal but I will double check with Dan about the file removal.

Please anyone looking at this PR be aware to look for any issues about this file being removed.